configure.ac

```diff
@@ -232,8 +232,8 @@
             RUST_LDADD="-ldl -lrt -lm"
             ;;
         *-*-mingw32*)
-            CFLAGS="${CFLAGS} -DOS_WIN32"
-            LDFLAGS="${LDFLAGS} -lws2_32"
+            CFLAGS="${CFLAGS} -DOS_WIN32 -DNTDDI_VERSION=NTDDI_VISTA -D_WIN32_WINNT=_WIN32_WINNT_VISTA"
```

Comment:
should this be conditional based on whether windivert is enabled?

Reply:
Good question. It _can_ be conditional, but the WMI offload query/disable code requires Vista or newer. I suspect you still want XP compatibility if possible? I can probably flag the functionality in an `#ifdef` on `NTDDI_VERSION`/`_WIN32_WINNT`, and have `--enable-windivert` switch those on by default.

Reply:
I don't specifically care about XP. Whats the lowest OS version still supported by Microsoft for the general public?

Reply:
Windows 7 is the oldest supported now. Windows Vista support technically ended a year ago.

I made the change to make this conditional testing another issue, so it'll be optional without WinDivert.